src/leetCodeExecutor.ts

```diff
@@ -34,7 +36,11 @@ class LeetCodeExecutor implements Disposable {
         return `"${path.join(this.leetCodeRootPath, "bin", "leetcode")}"`;
     }
 
-    public async meetRequirements(): Promise<boolean> {
+    public async meetRequirements(context: ExtensionContext): Promise<boolean> {
+        const isUserFresh: boolean | undefined = context.globalState.get(globalStateLeetcodeIsUserFresh);
+        if (isUserFresh !== false) {
+            await this.removeOldCache();
```

Comment:
Reuse the command we have?

https://github.com/LeetCode-OpenSource/vscode-leetcode/blob/master/src/extension.ts#L51

Reply:
Seems a little different.The command way use leetcode-cli way not delete the whole dir, let me check if it works fine too.

Reply:
@jdneo 
delete cache only delete the user session but not the plugin.js cache, still have that issue. So I think we can not use it..
![image](https://user-images.githubusercontent.com/15976103/88164168-5986b980-cc46-11ea-8753-ad8f2b7a2c5a.png)

Reply:
Maybe we need to change the implementation of the command `delete cache`. Considering that the current implementation does not clean entriely?

Reply:
Yes, current one only clean the user info(include session),  but we change the `delete cache` also have one problem beacuse user may have two account one leetcode one leetcode-cn, if we change that. both will clean, but user may only want to clean the current cache.

Reply:
I see. So let's keep the implementation. Maybe later we can provide two option in this command. Something like `Session` and `All`. Anyway it's not a high priority